server/plugin/plugin.go

```diff
@@ -990,3 +990,10 @@ func (p *Plugin) getUsername(mmUserID string) (string, error) {
 
 	return "@" + info.GitHubUsername, nil
 }
+
+func (p *Plugin) getSiteURL() string {
+	if p.API.GetConfig().ServiceSettings.SiteURL != nil {
+		return *p.API.GetConfig().ServiceSettings.SiteURL
+	}
+	return ""
+}
```

Comment:
Would you mind updating the rest of the code base, like in https://github.com/mattermost/mattermost-plugin-github/blob/c7124749f2af0a3c5060478ae26ce12b8bddabc4/server/plugin/api.go#L845?